src/components/popover/PopoverContent.tsx

```diff
@@ -17,7 +17,7 @@ export const PopoverContent = (props: HTMLProps<HTMLDivElement> & PopoverContent
       <FloatingFocusManager context={floatingContext}>
         <StyledPopoverContent
           $color={context.color}
-          border={1}
+          border={props.border ?? 0}
```

Comment:
With type set as before, we can set the value as:
```suggestion
          border={props.border}
```
which doesn't set any value in the end (equivalent to no border)